.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: RELEASE
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  repository_dispatch:
+    types: [release]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  new-version:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Check out repo
+        uses: actions/checkout@v2
+      # Node is required for npm
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      # Install and build Docusaurus website
+      - name: Build Docusaurus website
+        run: |
+          yarn 
+          yarn build
+      # https://docusaurus.io/docs/versioning#tagging-a-new-version
+      - name: Tag a new version, commit the changes and push
+        run: |
+          yarn run docusaurus docs:version ${{ github.event.client_payload.version }}
+          git config --local user.email "iasql-bot@users.noreply.github.com"
+          git config --local user.name "iasql-bot"
+          git commit -m "${{ github.event.client_payload.version }}" -a
+          git push
```

Comment:
So, newer versions of `git` won't auto-push all branches without specifying them or enabling a backwards compatibility flag, but I don't know how github has configured their own git, so let's see what this does?